cmd/livepeer/livepeer.go

```diff
@@ -204,47 +246,36 @@ func main() {
 						os.Exit(1)
 					}
 				}()
+				gethipc := fmt.Sprintf("%v/geth.ipc", defaultEthDatadir)
+				//Wait for gethipc
+				if _, err := os.Stat(gethipc); os.IsNotExist(err) {
+					start := time.Now()
+					glog.V(0).Infof("Waiting to start go-ethereum")
+					for time.Since(start) < time.Second*5 {
+						if _, err := os.Stat(gethipc); os.IsNotExist(err) {
+							time.Sleep(time.Millisecond * 500)
+						} else {
+							continue
```

Comment:
Should this be `break` instead of `continue`? Otherwise, wouldn't this loop always take the full 5 seconds since the loop will continue each time `os.Stat(gethipc)` does not return an error?